Sources/PrimerSDK/Classes/Core/Analytics/AnalyticsService.swift

```diff
@@ -247,6 +247,7 @@ extension Analytics {
             }
         }
 
+		// swiftlint:disable:next nesting
```

Comment:
tabs have come back in this file :-(

Reply:
<img width="456" alt="Screenshot 2024-02-06 at 10 47 55" src="https://github.com/primer-io/primer-sdk-ios/assets/137198782/33032c2f-1285-40f1-a7d7-775dcfebfaee">
I have no idea why is this happening to be honest... 

These are my Xcode settings, any ideas?
<img width="574" alt="Screenshot 2024-02-06 at 10 48 41" src="https://github.com/primer-io/primer-sdk-ios/assets/137198782/f9b3f0b5-72f7-4df3-b80f-b6a68f98166f">

Reply:
These are correct for Xcode afaik 

Did you definitely make these change in Xcode? Any other text editor? or could swiftlint fix have done this?

Reply:
I think swiftlint fix is doing that, but it is visible only in GitHub...